x-pack/plugins/aiops/public/components/change_point_detection/change_points_table.tsx

```diff
@@ -126,6 +167,63 @@ export const ChangePointsTable: FC<ChangePointsTableProps> = ({
             truncateText: false,
             sortable: true,
           },
+          {
+            name: i18n.translate('xpack.aiops.changePointDetection.actionsColumn', {
+              defaultMessage: 'Actions',
+            }),
+            actions: [
+              {
+                name: i18n.translate(
+                  'xpack.aiops.changePointDetection.actions.filterForValueAction',
+                  {
+                    defaultMessage: 'Filter for value',
+                  }
+                ),
+                description: i18n.translate(
+                  'xpack.aiops.changePointDetection.actions.filterForValueAction',
+                  {
+                    defaultMessage: 'Filter for value',
+                  }
+                ),
+                icon: 'plusInCircle',
+                color: 'primary',
+                type: 'icon',
+                onClick: (item) => {
+                  if (!item.group) return;
```

Comment:
Will it be clear what happened for a user if they click and just nothing happens? Maybe it makes sense to have this check as part of toggling the enabled/disabled state of the action and a tooltip with the reason.

Reply:
initially, I did it to make TypeScript happy, but this condition will never be truthy. There is a manual type casting at the next line, so I removed it here https://github.com/elastic/kibana/pull/155256/commits/b85027f2bf7fcbf0bd59d595ea6c05476438eea0